src/dashboard/Data/Browser/AddColumnDialog.react.js

```diff
@@ -54,11 +92,92 @@ export default class AddColumnDialog extends React.Component {
     );
   }
 
+  async handleDefaultValueChange(defaultValue) {
+    const { type, target } = this.state
+    let formattedValue = undefined
+    let isDefaultValueValid = true
+
+    try {
+      switch (type) {
+        case 'String':
+          formattedValue = defaultValue.toString()
+          break
+        case 'Number':
+          if (!validateNumeric(defaultValue)) throw 'Invalid number'
+          formattedValue = +defaultValue
+          break
+        case 'Array':
+          if (!Array.isArray(JSON.parse(defaultValue))) throw 'Invalid array'
+          formattedValue = JSON.parse(defaultValue)
+          break
+        case 'Object':
+          if (typeof JSON.parse(defaultValue) !== 'object' || Array.isArray(JSON.parse(defaultValue))) throw 'Invalid object'
+          formattedValue = JSON.parse(defaultValue)
+          break
+        case 'Date':
+          formattedValue = { __type: 'Date', iso: new Date(defaultValue) }
+          break
+        case 'Polygon':
+          formattedValue = new Parse.Polygon(JSON.parse(defaultValue))
+          break
+        case 'GeoPoint':
+          formattedValue = new Parse.GeoPoint(JSON.parse(defaultValue))
+          break;
+        case 'Pointer':
+          formattedValue = await this.handlePointer(defaultValue, target)
+          break
+        case 'Boolean':
+          formattedValue = (defaultValue === 'True' ? true : (defaultValue === 'False' ? false : undefined))
+          break
+        case 'File':
+          formattedValue = await this.handleFile(defaultValue)
+          break
+      }
+    } catch (e) {
+      isDefaultValueValid = defaultValue === ''
+    }
+    return await this.setState({ defaultValue: formattedValue, isDefaultValueValid })
+  }
+
+  renderDefaultValueInput() {
+    const { type } = this.state
+    switch (type) {
+      case 'Array':
+      case 'Object':
+      case 'Polygon':
+      case 'GeoPoint':
+        return <TextInput
+          placeholder='Set here a default value'
+          multiline={true}
+          onChange={async (defaultValue) => await this.handleDefaultValueChange(defaultValue)} />
+      case 'Number':
+      case 'String':
+      case 'Pointer':
+        return <TextInput
+          placeholder={type === 'Pointer' ? 'Set here a valid object ID' : 'Set here a default value'}
```

Comment:
```suggestion
          placeholder={type === 'Pointer' ? 'Set a valid object ID here' : 'Set a default value here'}
```

Reply:
@TomWFox thanks for the review! All suggestions were applied.